03.asynchronous/callback/callback_no_error.js

```diff
@@ -0,0 +1,25 @@
+import sqlite3 from "sqlite3";
+const db = new (sqlite3.verbose().Database)(":memory:");
+
+db.run(
+  "CREATE TABLE books (id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT NOT NULL UNIQUE )",
+  () => {
+    db.run(
+      "INSERT INTO books(title) VALUES (?)",
```

Comment:
スタイルを一貫させるならスペースがあったほうがよいと思います。

```suggestion
      "INSERT INTO books (title) VALUES (?)",
```

Reply:
スペースありに統一いたします。

Reply:
[余分なスペースの削除、空白の追加](https://github.com/zamami/js-practices/pull/3/commits/e72186f2041f4c8aa2242651de5bdfa59012c9d0)
こちらのコミットでスペースを追加しました。